code/modules/hydroponics/grown/mushrooms.dm

```diff
@@ -109,6 +110,37 @@
 	filling_color = "#DAA520"
 	wine_power = 80
 
+// Brig Cap
+/obj/item/seeds/liberty/brig
+	name = "pack of brig-cap mycelium"
+	desc = "The warden's number 1 choice."
+	icon_state = "mycelium-brig"
+	species = "brig"
+	plantname = "Brig-Caps"
+	product = /obj/item/reagent_containers/food/snacks/grown/mushroom/brigcap
+	maturation = 7
+	production = 1
+	yield = 2
+	potency = 15
+	growthstages = 3
+	genes = list(/datum/plant_gene/trait/plant_type/fungal_metabolism)
+	growing_icon = 'icons/obj/hydroponics/growing_mushrooms.dmi'
+	reagents_add = list( /datum/reagent/consumable/nutriment = 0.02,/datum/reagent/peaceborg/tire=0.1)
+	rarity = 30
+
+/obj/item/reagent_containers/food/snacks/grown/mushroom/brigcap
+	seed = /obj/item/seeds/liberty/brig
+	name = "brig-cap"
+	desc = "<B>Like it or not you're comming with me!</B>"
+	icon_state = "brigcap"
+	filling_color = "#DAA520"
+	wine_power = 80
+	distill_reagent = /datum/reagent/consumable/ethanol/beepsky_smash
+/obj/item/reagent_containers/food/snacks/grown/mushroom/brigcap/attackby(obj/item/W, mob/user, params)
+	if(W.sharpness)
+		user.show_message("<span class='notice'>You make hancuffs out of \the [src]!</span>", 1)
+		new /obj/item/restraints/handcuffs(user.loc)
```

Comment:
FYI the variable is `breakouttime`, you could have these scale with potency (100 pot = same as normal handcuffs).